docs/current/guides/snippets/use-services/environment-variables/main.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import anyio
+import dagger
+
+def env_variables(envs):
+    def wrapper(ctr):
```

Comment:
Nit: add types and `wrapper` isn't correct as a name because it's not wrapping anything.
```suggestion
def env_variables(envs: dict[str, str]):
    def func(ctr: dagger.Container):
```